app/helpers/session_answers_helper.rb

```diff
@@ -16,13 +16,27 @@ def govuk_radio_for_node(form)
       "govuk_publishing_components/components/radio",
       heading: tag.h1(content_for(:title), class: "govuk-fieldset__heading"),
       heading_size: "xl",
+      hint: tag.span(content_for(:hint), class: "govuk-caption-l"),
       name: "#{form.node_name}[]",
       items: form.radio_options,
       error_message: error_summary(form, form.node_name),
       id: form.node_name,
     )
   end
 
+  def govuk_checkbox_for_node(form)
+    render(
+      "govuk_publishing_components/components/checkboxes",
+      heading: tag.h1(content_for(:title), class: "govuk-fieldset__heading"),
+      heading_size: "xl",
+      hint_text: tag.span(content_for(:hint), class: "govuk-caption-l"),
+      name: "#{node_name}[]",
+      items: @form.checkbox_options,
+      error: error_summary(@form, node_name),
+      id: node_name,
+    )
+  end
```

Comment:
I'd suggest creating a hint method something like:

```ruby
def hint_span
  return unless content_for?(:hint)
  
  tag.span(content_for(:hint), class: "govuk-caption-l")
end
```

Then the change on line 19 would be:
```ruby
hint: hint_span
```

And on line 32:
```ruby
hint_text: hint_span
```

That way empty spans won't be inserted if no hint is presents/required.